resources/assets/components/pages/ShowSubmissionPage/ShowSubmissionPage.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
 
+import Query from '../../Query';
+import { query } from '../../../helpers';
 import SiteFooter from '../../utilities/SiteFooter/SiteFooter';
+import TextContent from '../../utilities/TextContent/TextContent';
 import SiteNavigationContainer from '../../SiteNavigation/SiteNavigationContainer';
+import { CONTENTFUL_BLOCK_QUERY } from '../../utilities/ContentfulEntryLoader/ContentfulEntryLoader';
 
 const ShowSubmissionPage = () => {
+  const id = query('submissionActionId');
+  const defaultContent =
+    'Thanks for joining the movement! After we review your submission, we&apos;ll add it to the public gallery alongside submissions from all the other members taking action in this campaign.';
   return (
```

Comment:
Nit: adding a newline before `return`
```suggestion

  return (
```